web_src/css/markup/content.css

```diff
@@ -144,7 +144,7 @@
 
 .markup ul,
 .markup ol {
-  padding-left: 2em;
+  padding-left: 0;
```

Comment:
@rafh I'm worried about this style affecting other elements unrelated to checkboxes. Are there other `ul` `ol` items this will apply to in markdown (I assume yes)?

Especially because this value was not changed here: https://github.com/go-gitea/gitea/pull/17411/files

(btw, this is an area front-end testing could validate)

Reply:
@kdumontnu I understand your point. Instead, we should change the `em` back to `2em` and update the `.task-list-item` to have `margin-left: -2em;`. This will ensure that all checkboxes are aligned properly without impacting all `ol` and `ul`.

Here's an example of the markup and how it is rendered within Gitea.

```MD
- [x] example content

<img  align="left" alt="" width="200" src="https://user-images.githubusercontent.com/6152817/265531274-d0b0b99a-db2b-4b6d-946d-277bfd24fc11.jpg">

- [x] example content

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed efficitur orci, eu cursus sapien. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed efficitur orci, eu cursus sapien. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed efficitur orci, eu cursus sapien. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed efficitur orci, eu cursus sapien. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur sed efficitur orci, eu cursus sapien.

<img style="clear: both" alt="" width="200" src="https://user-images.githubusercontent.com/6152817/265531274-d0b0b99a-db2b-4b6d-946d-277bfd24fc11.jpg">


- [ ] one

- [ ] two
```

<img width="783" alt="Screenshot 2023-09-07 at 2 51 27 PM" src="https://github.com/go-gitea/gitea/assets/6152817/e93397e7-696c-4558-b84d-c99e427bf1e4">

Reply:
That seems like a reasonable solution to me, or maybe you can add a `:not(.task-list-item)` to the selectors above and. Whatever will apply the style to the `.task-list-item` items, but not change the styling for other lists.